wdl/model/draft2/src/main/scala/wdl/draft2/model/expression/WdlStandardLibraryFunctions.scala

```diff
@@ -325,7 +331,20 @@ object WdlStandardLibraryFunctions {
 
         // Inner function: Get the file size, allowing for unpacking of optionals
         def optionalSafeFileSize(value: WomValue): Try[Long] = value match {
-          case f if f.isInstanceOf[WomSingleFile] || WomSingleFileType.isCoerceableFrom(f.womType) => Try(Await.result(ioFunctionSet.size(f.valueString), Duration.Inf))
+          case f if f.isInstanceOf[WomSingleFile] || WomSingleFileType.isCoerceableFrom(f.womType) => {
+            val uuid = UUID.randomUUID().toString
```

Comment:
TOL: For this and the other UUIDs in this PR, is there a way we could format them differently so they're harder to confuse with other UUIDs in the logs?

Reply:
I put `()` around them. Very different than the others surrounded by `[]`. 😉 

If you have some other prefix/suffix suggestion I'm happy to swap it out.